_topic_maps/_topic_map.yml

```diff
@@ -922,7 +922,7 @@ Topics:
 - Name: Configuring identity providers
   Dir: identity_providers
   Topics:
-  - Name: Configuring an HTPasswd identity provider
```

Comment:
split the changes where you're making HTPasswd IdP lowercased to a separate commit.